src/Adapters/Storage/Mongo/MongoSchemaCollection.js

```diff
@@ -274,6 +274,20 @@ class MongoSchemaCollection {
         }
       });
   }
+
+  async updateFieldOptions(className: string, fieldName: string, fieldType: string) {
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Why do we need this?

Reply:
@mtrezza can i close this?   If not, can you propose a solution ?

Reply:
We should disable lint only if justified, and I am still not sure if that's the case here. Why are extracting these variables if they are unused? We can just not extract them?